llvm/lib/Target/PowerPC/PPCInstrFormats.td

```diff
@@ -725,6 +725,38 @@ class XForm_24_sync<bits<6> opcode, bits<10> xo, dag OOL, dag IOL,
   let Inst{31}    = 0;
 }
 
+class XForm_24_P10wait<bits<6> opcode, bits<10> xo, dag OOL, dag IOL,
```

Comment:
Most of these classes don't have the instruction name ref in it.  Any specific reason why these do instead of following the newly setup instr class naming convention?